ec2Bootstrap/userData.sh

```diff
@@ -33,14 +35,31 @@ apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv 2EE0EA64E40A89B84B2
 apt-get update
 apt_get_install sbt
 
+# Install AWS cli
+apt-get_install awscli
+
 mkdir /opt/elasticsearch-migration
 cd /opt/elasticsearch-migration
-wget https://github.com/NewsWhip/es-actors/archive/v1.3.9.tar.gz
-tar -xvf v1.3.9.tar.gz
-cd es-actors-1.3.9
-SERVER_CMD="sbt -J-Xmx30G -J-Xms30G \"; project server; run-main com.broilogabriel.Server\""
+# first setup Spike -> Analytics
+wget https://github.com/NewsWhip/es-actors/archive/v$ES_ACTORS_VERSION.tar.gz
+tar -xvf v$ES_ACTORS_VERSION.tar.gz
+cd es-actors-$ES_ACTORS_VERSION/es-actors
+SERVER_CMD="sbt -J-Xmx25G -J-Xms25G "project server" run"
```

Comment:
It's needed to use the \ char to make it work:
`SERVER_CMD="sbt -J-Xmx25G -J-Xms25G \"project server\" run"`